src/stream/src/executor/source/executor_core.rs

```diff
@@ -59,6 +63,9 @@ where
         source_desc_builder: SourceDescBuilder,
         split_state_store: SourceStateTableHandler<S>,
     ) -> Self {
+        let is_batch_source = source_desc_builder
+            .with_properties()
+            .is_refreshable_connector();
```

Comment:
Can we try unifying the terminology (batch vs refreshable)?

Reply:
Or just "refreshable batch source"?

Reply:
Or "batch source" + "refreshable table"?